src/endpoint.ts

```diff
@@ -65,9 +77,10 @@ export abstract class AbstractEndpoint {
   ): string | undefined;
   public abstract getMethods(): Method[];
   public abstract getSchema(variant: IOVariant): IOSchema;
-  public abstract getSchema(variant: ResponseVariant): z.ZodTypeAny;
-  public abstract getMimeTypes(variant: MimeVariant): string[];
-  public abstract getStatusCode(variant: ResponseVariant): number;
```

Comment:
⚠️ potentially breaking (however, consumers are not supposed to use those methods)